microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -204,4 +204,13 @@ Issued: 8 January 2024
 
 {product-title} release 4.17.11 is now available. Bug fixes and enhancements are listed in the link:https://access.redhat.com/errata/RHBA-2025:0025[RHBA-2025:0025] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHBA-2025:0023[RHBA-2025:0023] advisory.
 
-See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
\ No newline at end of file
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="microshift-4-17-12-dp_{context}"]
+=== RHBA-2025:0117 - {microshift-short} 4.17.12 bug fix and enhancement advisory
+
+Issued: 14 January 2024
```

Comment:
If so, the previous two releases from January also show 2024.
```suggestion
Issued: 14 January 2025
```

Reply:
Shazbat!